README.md

```diff
@@ -141,21 +146,25 @@ to install the dependencies.
 
 ### Debugging
 
-You can debug tests by using `bazel run` and passing `--node_options=--inspect`. For example, to
-debug a specific golden test:
+You can debug tests by using `bazel run` and passing `--node_options=--inspect`
+or `--node_options=--inspect-brk` (to suspend execution directly after startup).
+
+For example, to debug a specific golden test:
 
 ```shell
-TEST_FILTER=my_golden_test ibazel run //test:golden_test -- --node_options=--inspect
+TEST_FILTER=my_golden_test ibazel run //test:golden_test -- --node_options=--inspect-brk
 ```
 
-Then open [about:inspect] in Chrome and choose "about:inspect". Chrome will launch a debugging
-session on any node process that starts with a debugger listening on one of the listed ports. The
-tsickle tests and Chrome both default to `localhost:9229`, so things should work out of the box.
+Then open [about:inspect] in Chrome and choose "about:inspect". Chrome will
+launch a debugging session on any node process that starts with a debugger
+listening on one of the listed ports. The tsickle tests and Chrome both default
+to `localhost:9229`, so things should work out of the box.
+
+The break in specific code locations you can add `debugger;` statements in the
+source code.
 
-VS Code can also connect using the inspect protocol. It doesn't support automatically reconnecting
-or any way to re-run the test suite though, so it is a less convenient. You can start the node
-process passing an extra `--node_options=--debug-brk` (in addition to the parameters above) to have
-Node wait before program execution, so you have time to attach VS Code.
+Note: IDEs such as VS Code have support for the inspect protocol, but the
+integration does not work due to bazel's complex directory layout.
```

Comment:
This is fine, but if you wanted to tinker with it more I think you might need to run `nodejs --preserve-symlinks` .  That at least fixed another thing I found when tinkering with bazel.

Reply:
Yes we always run `node --preserve-symlinks --preserve-symlinks-main` - the latter avoids the entry point script getting `realpath`ed

Reply:
The issue is that when we launch node, we've already resolved all the paths to absolute locations that are in some `$TMP/bazel-cache` directory. At that point source map resolution appears impossible, as Alex says.